src/mono/wasm/debugger/DebuggerTestSuite/MiscTests.cs

```diff
@@ -928,5 +928,15 @@ public async Task InspectLocalsUsingClassFromLibraryUsingDebugTypeFull()
             );
         }
         //TODO add tests covering basic stepping behavior as step in/out/over
+
+        [Fact]
+        public async Task SetBreakpointInProjectWithSpecialCharactersInPath()
```

Comment:
It would be useful to check a path having non-ascii chars too.
And checking a source file name with such characters.

Reply:
I added both and still working.